packages/integrations/src/vue.ts

```diff
@@ -1,103 +1,298 @@
 import { EventProcessor, Hub, Integration } from '@sentry/types';
-import { getGlobalObject, isPlainObject, logger } from '@sentry/utils';
+import { basename, getGlobalObject, logger, timestampWithMs } from '@sentry/utils';
+import { Integrations as APMIntegrations, Span as SpanClass } from '@sentry/apm';
+
+interface IntegrationOptions {
+  Vue: any;
```

Comment:
Would it make sense to define a type based on the attributes we expect from the `Vue` object instead of `any`?

E.g. we access `Vue.mixin`, `Vue.config`, `Vue.util`, ...

Reply:
Sure, although it'd still be all "fake" and I'm not sure if we gain anything from it. We don't use official Vue types because it'd make everyone doing npm install @sentry/integrations fetching them.